### DISCUSSION TITLE
What does the following exactly mean ? `-o, --offset <offset>`

### DISCUSSION THREAD

**Discussion by TRaSH-** [cross-seed/cross-seed#68] [Q&A] [answered]
What does the following exactly mean ? `-o, --offset <offset>` Offset to start from

**Comment by mmgoodnow** [comment, marked as answer]
If you were running cross-seed previously and it crashed on the `n`th torrent, then you can set `--offset n` to resume where you left off.

**Reply by ajhorvath20** [reply]
That means if I restart the container then cross-seed will pick up from where it was shutdown? 

I need to add this in my compose, do I need to put `n` in there or something else. Because I cannot see it in docs.

`command: search offset n`

Thank you

**Reply by mmgoodnow** [reply]
`search --offset n`

Well you need to put a number there, you can't just put "n" and expect it to know where it last failed.

**Reply by ajhorvath20** [reply]
Thanks for the quick response but I'm confused with offset option because if I have 10,000 torrents and I have offset set to 600 and I restarted the cross-seed at 400 so that means cross-seed will start working from 600 and ignore anything between 400-600? I thought it will start automatically from last search where it crashed. Unless I'm all wrong :)

**Reply by mmgoodnow** [reply]
Offset should not be used in any automatic runs, it doesn't fit the use case. 

There is no logic that makes cross-seed resume where it left off last time.

**Reply by ajhorvath20** [reply]
That completely make sense. Thank you so much. Much appreciated for clarification :)
